WordPress/WordPressTest/WPCrashLoggingDataProviderTests.swift

```diff
@@ -0,0 +1,79 @@
+import XCTest
+import AutomatticTracks
+
+@testable import WordPress
+
+final class WPCrashLoggingDataProviderTests: XCTestCase {
+
+    // MARK: - Testing Log Error
+
+    func testReadingTrackUserInMainThread() throws {
+        // Given
+        let dataProvider = self.makeCrashLoggingDataProvider()
+
+        // When
+        let user = try XCTUnwrap(dataProvider.currentUser)
+
+        // Then
+        XCTAssertEqual(user.userID, "\(Constants.defaultAccountID)")
+        XCTAssertEqual(user.username, Constants.defaultAccountUsername)
+        XCTAssertEqual(user.email, Constants.defaultAccountEmail)
+    }
+
+    func testReadingTrackUserInBackgroundThread() {
```

Comment:
Use the same test body as the above function but inside an `async` function probably is enough to cover the scenario where `currentUser` is called from a background thread.

However, if you do want to test concurrency calls to the `currentUser` function, using a `TaskGroup` probably would be easier than dispatch group. Something like this:

``` throws
func testReadingTrackUserInBackgroundThread() async {
    await withTaskGroup(of: Void.self) { group in
        for _ in 1...1000 {
            group.addTask {
                let user = dataProvider.currentUser
                XCTAssert(...)
            }
        }
    }
    // No assertion or waiting needed at the end.
}
```